docs/articles/string_concatenation.md

```diff
@@ -0,0 +1,11 @@
+﻿# String concatenation
+
+If the both left and right values of expression are string the result will be a string concatenation. If not, you can still concatenate values as a string by using <xref:NCalc.ExpressionOptions.StringConcat> option.
+
+```c#
+try
+{
+   var expression = new Expression($"5 +'1'", ExpressionOptions.StringConcat);
+}
+catch (OverflowException ex) { }
```

Comment:
Sorry for the small nitpick, but I think this catch is not needed. I would improve this article with the example of the PR
``` 
'L' + 'eo' -> Always concat regardless of StringConcat
1 + '2' -> Same behavior as old versions with StringConcat off and '12' with StringConcat on
'1' + '2' -> 3 with StringConcat off and '12' with StringConcat on
```